scripts/check_codebase_language.py

```
@@ -0,0 +1,177 @@
+
+
+def main():
+    """Run LanguageTool checks on the codebase."""
+    print("Checking codebase with LanguageTool...")
+
+    # Initialize LanguageTool (use local server only)
+    tool = None
+    try:
+        tool = language_tool_python.LanguageTool("en-US")
+        print("Using LanguageTool local server")
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)